src/data_processing/ukhra_data.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+from rich.progress import track
+
+
+def read_ukhra_dataset(year):
+    """Read UKHRA xlsx dataset and clean columns.
+
+    Args:
+        year(int): Year of UKHRA study.
+
+    Returns:
+        pd.DataFrame: UKHRA dataframe for given year.
+
+    """
+    ukhra_df = pd.read_excel(
+        f'data/raw/ukhra{year}.xlsx',
```

Comment:
It might be better to feed this as an argument in case either the directory changes or you call this definition from a different working directory.